CDCgov/prime-reportstream
# Onboarding AK setting

This PR adds onboard settings and test files for AK.

Test Steps:
1. ./gradlew clean package
2. ./gradlew reloadTables
3. ./prime fhirdata --input-file /home/judith/workspace/cdc/working/prime-reportstream/prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_SR_2_20231106-0003.fhir -s metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml --output-format HL7 -r metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml --output-file src/testIntegration/resources/datatests/FHIR_to_HL7/AK-TEST.hl7
4. Examine the AK-TEST.hl7 output file and make sure you see the following:
. MSH-6: AK-PHD
. PID-11: Patient Address:                      
  PID-11-1:                                   5555 Street
  PID-11-3:                                   City
  PID-11-4:                                   AK
  PID-11-5:                                   55555
  PID-11-6:                                   USA
. PID-22-3: CDCREC
. PID-5-7, ORC-12-1, OBR-16-1 are empties.

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
